src/main/java/com/ust/sourcecourse/configuration/excpection/ResourceNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.ust.sourcecourse.configuration.excpection;
```

Comment:
Correct the package name. Take latest code from dev. There is already ResourceNotFoundException class added, but in exceptions package. You can use the same and delete this package.